File: Tamago-FrontEnd/src/app/Models/FoodModel.ts
export class FoodModel {
  public id: String;
  public name: String;

  public price: number;
  public discount: number;

  public category: String;

  public foodVal: number;
  public energyVal: number;
  public happyVal: number;

  constructor(id, name, price, discount, category, foodVal, energyval, happyVal) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.discount = discount;
    this.category = category;
    this.foodVal = foodVal;
    this.energyVal = energyval;
    this.happyVal = happyVal;
  }
}

File: Microservices Demo/Shared/ApiModels/FoodModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Shared.Shared.ApiModels
{
	public class FoodModel
	{
		public Guid id { get; set; }

		public string name { get; set; }
		public string category { get; set; }

		public double price { get; set; }
		public double discount { get; set; }

		public double foodVal { get; set; }
		public double energyVal { get; set; }
		public double happyVal { get; set; }
	}
}
